src/index.html

```diff
@@ -22,6 +22,335 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="images/imac.jpeg"
+          alt="imac"
+          class="picture"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__title_code">Product code: 195434</p>
+        <div class="marks">
+          <div class="marks_stars">
+            <div class="marks_star"></div>
+            <div class="marks_star"></div>
+            <div class="marks_star"></div>
+            <div class="marks_star"></div>
+            <div class="marks_star"></div>
+          </div>
```

Comment:
Stars must be a separate BEM block, fix it 
```suggestion
          <div class="stars stars--4">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>
```